calico_node/startup/startup.go

```diff
@@ -831,7 +831,7 @@ func ensureDefaultConfig(ctx context.Context, cfg *apiconfig.CalicoAPIConfig, c
 			newFelixConf.Spec.ReportingIntervalSecs = &interval
 			newFelixConf.Spec.LogSeverityScreen = defaultLogSeverity
 			_, err = c.FelixConfigurations().Create(ctx, newFelixConf, options.SetOptions{})
-			if err != nil {
+			if checkConflictError(err) != nil {
```

Comment:
how would you get a resourceUpdateConflict error for a `Create()`? Create doesn't have a resourceVersion so it can't have an update conflict

Reply:
Maybe we can check for `ResourceAlreadyExists` error, but then that's not something we want to ignore since we won't be able to update the fields once set by a node

Reply:
So, what seems to happen is that two nodes try to set the same global value, one node succeeds after the other node has already called get but before it creates, then it tried to create and there is already a value there and a resourceUpdateConflict is thrown.

Reply:
Are you sure? It should throw a `ResourceAlreadyExists` error if another node gets to it first, `Create` call shouldn't generate that error

Reply:
I'm mostly sure :) I'll do some more digging to verify.

Reply:
I wasn't able to get it to specifically fail on that error, but I agree with you on this. I've added another check in the `checkConflictError` function to check if the error is of type `ErrorResourceAlreadyExists` as well.